Sources/MapboxDirections/RestStop.swift

```diff
@@ -11,7 +10,10 @@ public struct RestStop: Codable, Equatable {
         case restArea = "rest_area"
     }
 
-    let type: StopType
+    /**
+     The kind of the rest stop.
+     */
+    public let type: StopType
```

Comment:
Similarly, `RestStop.StopType` is redundant. Let’s use `Kind` whenever possible.